cairo/tests/utils/strategies.py

```diff
@@ -290,26 +291,33 @@ def tuple_strategy(thing):
 
 # Fork
 state = st.lists(bytes20, min_size=0, max_size=MAX_ADDRESS_SET_SIZE).flatmap(
-    lambda addresses: st.fixed_dictionaries(
-        {
-            "_main_trie": st.builds(
-                lambda data: Trie(secured=True, default=None, _data=data),
-                data=st.fixed_dictionaries(
-                    {address: st.from_type(Account) for address in addresses}
-                ),
-            ),
-            "_storage_tries": st.fixed_dictionaries(
-                {
-                    address: st.builds(
-                        lambda data: Trie(secured=True, default=0, _data=data),
-                        data=st.dictionaries(bytes32, uint256),
-                    )
-                    for address in addresses
-                },
+    lambda addresses: st.builds(
+        State,
+        _main_trie=st.builds(
+            Trie,
+            secured=st.just(True),
+            default=st.none(),
+            _data=st.fixed_dictionaries(
+                {address: st.from_type(Optional[Account]) for address in addresses}
```

Comment:
Account has code which are bytes and can be huge if not controlled.

Reply:
should we build an account strategy in a subsequent pr?

Reply:
maybe indeed